### PR TITLE
notion-md-gen 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by saltbo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/notion-md-gen.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      tag:      "v1.2.0",
+      revision: "6db355b7e63ba8b0b745989f84ff42b8c65f4637"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags",
+      "-s -w -X notion-md-gen/cmd.release=#{version} -X notion-md-gen/cmd.commit=#{Utils.git_head} -X notion-md-gen/cmd.repo=#{stable.url}",
+      *std_go_args,
+      "./main.go"
```
Comment (carlocab) [inline Formula/notion-md-gen.rb:14]:
```suggestion
    ldflags = %W[
      -s -w
      -X notion-md-gen/cmd.release=#{version}
      -X notion-md-gen/cmd.commit=#{Utils.git_head}
      -X notion-md-gen/cmd.repo=#{stable.url} 
    ]
    system "go", "build", *std_go_args(ldflags: ldflags), "./main.go"
```

File: Formula/notion-md-gen.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/notion-md-gen version 2>&1")
```
Comment (carlocab) [inline Formula/notion-md-gen.rb:18]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

File: Formula/notion-md-gen.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+class NotionMdGen < Formula
+  desc "Convert notion content to markdown for static blog"
+  homepage "https://github.com/saltbo/notion-md-gen"
+  url "https://github.com/saltbo/notion-md-gen.git",
+      tag:      "v1.2.0",
+      revision: "6db355b7e63ba8b0b745989f84ff42b8c65f4637"
```
Comment (carlocab) [inline Formula/notion-md-gen.rb:6]: This needs a `license` declaration.

**Review by carlocab — COMMENTED** [review]
Please checkout your PR branch and run
```
brew audit --new notion-md-gen
```
and address the resulting errors.

**Comment by saltbo** [comment]
wait some days

**Comment by carlocab** [comment]
Thanks for the PR. This looks like a nice piece of software, but it currently does not meet the requirements for inclusion into Homebrew/core. Running `brew audit --new notion-md-gen` as requested in the PR template reads:
```
  notion-md-gen:
    * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```
You are welcome to submit this again when it meets one of the above requirements.

If you wish, you can host this formula in your own tap in the meantime. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Comment by saltbo** [comment]
thanks, I will back later.

And, I don't know why I don't have any hints when I execute `brew audit --new notion-md-gen`.

**Comment by carlocab** [comment]
You might have passed a path to the formula (e.g. `brew audit --new ./notion-md-gen`) instead of the name of the formula (`brew audit --new notion-md-gen`).

**Pull request closed without merging**